runtime/flags.go

```diff
@@ -36,6 +36,18 @@ var Flags = []cli.Flag{
 		Name:     "runtime.namespace",
 		Usage:    "namespace to use for the runtime (only used by kubernetes)",
 	},
+	&cli.StringFlag{
+		EnvVars:  []string{"VELA_RUNTIME_PODS_TEMPLATE_NAME", "RUNTIME_PODS_TEMPLATE_NAME"},
+		FilePath: "/vela/runtime/pods-template-name",
+		Name:     "runtime.pods-template-name",
```

Comment:
nit: i know we're not super consistent for the `Name` field here, but we do seem to settle on using `_` instead of `-` for `FilePath` values. But as `Name` is really only used internally, it's easier to change after the fact. `FilePath` not so much.

Reply:
Yeah, I was honestly wondering if `pods` should be `runtime.pods` since `pods` have multiple things under it but idk you're right about the lack of consistency.

Reply:
true - we can add consistency to `Name` outside of this PR, but that should probably be a thing :)

Reply:
OK - so I think you're suggesting this change for FilePath:
```suggestion
		FilePath: "/vela/runtime/pods_template_name",
		Name:     "runtime.pods-template-name",
```

I don't understand the issue with the `Name` field however.
Are you suggesting we use `runtime.pods.template-name` and `runtime.pods.template-file`?

Reply:
I made the `FilePath` change.